docs/google-kubernetes-tutorial.md

```diff
@@ -0,0 +1,144 @@
+
+## Installing Helm
+
+Helm is the package manager for Kubernetes, and is what allows us to install all of the distributed components of TiDB in a single step.  Helm requires both a server-side and a client-side component to be installed.
+
+Because Cloud Shell system packages do not persist, we also copy the `helm` command to a local directory.
+
+Download the `helm` installer:
+
+	curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > get_helm.sh
+
+Install `helm`:
+
+	chmod 700 get_helm.sh && ./get_helm.sh
+
+Copy `helm` to a local directory:
+
+	mkdir -p ~/bin &&
+	cp /usr/local/bin/helm ~/bin &&
+	echo 'PATH="$PATH:$HOME/bin"' >> ~/.bashrc
+
+Helm will also need a couple of permissions to work properly:
+
+	kubectl create serviceaccount tiller --namespace kube-system &&
+	kubectl apply -f ./manifests/tiller-rbac.yaml &&
+	helm init --service-account tiller --upgrade
+
+It takes a minute for helm to initialize `tiller`, its server component:
+
+	watch "kubectl get pods --namespace kube-system | grep tiller"
+
+When you see `Running`, it's time to hit `Control + C` and proceed to the next step!
+
+## Deploy TiDB Operator
+
+The first TiDB component we are going to install is the TiDB Operator, using a Helm Chart.  The TiDB Operator is the management system that works with Kubernetes to bootstrap your TiDB cluster and keep it running. This step assumes you are in the `tidb-operator` working directory:
+
+	kubectl apply -f ./manifests/crd.yaml &&
+	kubectl apply -f ./manifests/gke-storage.yml &&
+	helm install ./charts/tidb-operator -n tidb-admin --namespace=tidb-admin
+
+We can watch the operator come up with:
+
+	watch kubectl get pods --namespace tidb-admin -o wide
+
+When you see `Running`, `Control + C` and proceed to launch a TiDB cluster!
+
+## Deploy your first TiDB Cluster
+
+Now with a single command we can bring-up a full TiDB cluster:
+
+	helm install ./charts/tidb-cluster -n tidb --namespace=tidb
+
+It will take a few minutes to launch.  You can monitor the progress with:
+
+	watch kubectl get pods --namespace tidb -o wide
+
+When you see all pods `Running`, it's time to `Control + C` and proceed forward!
+
+## Connecting to TiDB
+
+There can be a small delay between the pod being up and running, and the service being available.  You can watch list services available with:
+
+	watch "kubectl get svc -n tidb"
+
+When you see `demo-tidb` appear, you can `Control + C`.  The service is ready to connect to!
+
+You can connect to the clustered service within the Kubernetes cluster:
+
+	kubectl run -n tidb mysql-client --rm -i --tty --image mysql -- mysql -P 4000 -u root -h $(kubectl get svc demo-tidb -n tidb --output json | jq -r '.spec.clusterIP')
+
+For debugging purposes, you can also establish a tunnel between an individual TiDB pod and your Cloud Shell.  It is important to note that this will not survive if the pod `demo-tidb-0` fails our your Google Shell restarts.  For example:
```

Comment:
The last sentence is hard to understand.

Reply:
How about:
You can also establish a tunnel between an individual pod and your Cloud Shell.  This is recommended only for debugging purposes since the tunnel will not automatically be transferred if `demo-tidb-0` fails, or your Cloud Shell restarts.  To establish a tunnel:

Reply:
LGTM